app/src/types/payments/payment-method.ts

```diff
@@ -0,0 +1,63 @@
+declare module Stripe {
```

Comment:
What is the point of declaration as a module? To group them and "mark" as connected to stripe?

Reply:
Yes